### ISSUE TITLE
Getting ROOMS into shape to load, demo, and explore

### ISSUE THREAD

**Issue by masinter** [Interlisp/medley#909]
ROOMS is an application built by researchers at PARC to manage their windows and deal with managing screen space.  It was a 'first' (or early) desktop manager, built in Interlisp, separately patented (in the 1980's).

It was updated by ENvos and Venue and sold separately. The ideas are novel and not obviously intuitive, though. The documentation is scattered and unsorted.

I think what needs to happen is to sort out the files and documentation, find the papers and fix up the bibliography to talk about it, perhaps record some demos or even interview Austin Henderson, Doug Cutting and other inventors, and make it an option for online.interlisp.org.

**Comment by masinter** [comment]
@ekaltman @Morgan-AM ? a possibility?

**Comment by masinter** [comment]
I think all this needs is a little introduction, with links to the papers and the (now expired) patents and online documentation. ROOMS itself works fine, ahtough I had feared there might be some interactions between ROOMS and MODERNIZE, I haven't tested extensively, but so far no problems.

(FILESLOAD (FROM VALUEOF (MEDLEYDIR :ROOMS)) ROOMS   will load in all of the ROOMS files.
It changes your background pattern to say "Original" in the lower left corner and adds a Rooms menu item to the background Right click menu.  The documents about Rooms are in the Interlisp Zotero library (or should be), and an explanation of what it was and how it works would be useful.

Rooms uses a different cadence of operation, where the order of 'select object' vs 'perform operation' is odd for modern users -- so it takes a little time working through to understand.

(I raised this issue when I couldn't figure out simple operations ).
